src/app/config/tor_cmdline_mode.h

```diff
@@ -16,14 +16,16 @@
  * Enumeration to describe which command Tor is running.  These commands
  * are controlled by command-line options.
  **/
-typedef enum tor_cmdline_mode_t {
+typedef enum {
   CMD_RUN_TOR=0, /**< The default: run Tor as a daemon. */
   CMD_LIST_FINGERPRINT, /**< Running --list-fingerprint. */
   CMD_HASH_PASSWORD, /**< Running --hash-password. */
   CMD_VERIFY_CONFIG, /**< Running --verify-config. */
   CMD_DUMP_CONFIG, /**< Running --dump-config. */
   CMD_KEYGEN, /**< Running --keygen */
   CMD_KEY_EXPIRATION, /**< Running --key-expiration */
+  CMD_OTHER, /**< Special value: indicates a command that is handled
```

Comment:
Can we call this "CMD_IMMEDIATE"?
"OTHER" is not a very descriptive name.

Reply:
Okay; done in 432791c58f5d01